source-mongodb/main_test.go

```diff
@@ -806,3 +806,62 @@ func (v *correctnessInvariantsCaptureValidator) Summarize(w io.Writer) error {
 func (v *correctnessInvariantsCaptureValidator) bindingIdentifier(database, collection string) string {
 	return database + "." + collection
 }
+
+// TestNestedCursorFields tests nested cursor field lookups like "a.b".
+func TestNestedCursorFields(t *testing.T) {
+	const (
+		database   = "testDb"
+		collection = "nestedCursorCollection"
+	)
+	ctx := context.Background()
+	client, cfg := testClient(t)
+
+	cleanup := func() {
+		require.NoError(t, client.Database(database).Drop(ctx))
+	}
+	cleanup()
+	t.Cleanup(cleanup)
+
+	binding := makeBinding(t, database, collection, captureModeIncremental, "a.b")
+	cs := &st.CaptureSpec{
+		Driver:       &driver{},
+		EndpointSpec: &cfg,
+		Checkpoint:   []byte("{}"),
+		Validator:    &st.OrderedCaptureValidator{},
+		Sanitizers:   commonSanitizers(),
+		Bindings:     []*flow.CaptureSpec_Binding{binding},
+	}
+
+	col := client.Database(database).Collection(collection)
+	refTime := time.Date(2023, 11, 7, 16, 57, 0, 0, time.UTC)
```

Comment:
This `refTime` and related handling appears to be a superfluous holdover of copypasta from the timeseries tests.